### PR TITLE
[+] Add execution message

### PR DESCRIPTION
**Pull request by marikrebega**
image

added new example of usage to README
deploy_pin

### CODE REVIEW

File: lib/deploy_pin/runner.rb [hunk 1]
```diff
@@ -4,8 +4,14 @@
 module DeployPin
   module Runner
     def self.run(identifiers:)
-      DeployPin::Collector.new(identifiers: identifiers).run do |index, count, task, executable|
-        self.print("[#{index + 1}/#{count}] Task #{task.title} #{task.uuid}##{task.group} (#{executable ? 'run' : 'skip'})")
+      DeployPin::Collector.new(identifiers: identifiers).run do |index, count, task, executable, start:|
+        end_of_msg = if executable
+          start ? '(Run)' : "(Done)\n\n"
```
Comment (OrestOstashuk) [inline lib/deploy_pin/runner.rb:9]: (Run) -> maybe better (Started)

Comment (serhii-bodnaruk) [inline lib/deploy_pin/runner.rb:9]: let's add a message like this:
`Done in 30 seconds`

File: lib/deploy_pin/runner.rb [hunk 2]
```diff
@@ -4,8 +4,14 @@
...
+        else
+          "(Skip)\n\n"
```
Comment (OrestOstashuk) [inline lib/deploy_pin/runner.rb:11]: Skipped

File: lib/deploy_pin/runner.rb (hunk 1 above)
Comment (skcc321) [reply lib/deploy_pin/runner.rb:9]: would be great to create a formatter parameter for the task execution output.
you should be able to pass it as a second argument like ...run(identifiers:, formatter: ->(task attributes) { "the output message #{task_attributes[:execution_time]}" }
also please define default formatter

**Review by skcc321 — APPROVED** [review]

**Pull request merged**